src/main/java/org/jenkinsci/plugins/xunit/XUnitChecksPublisher.java

```diff
@@ -158,10 +158,6 @@ private String extractChecksTitle() {
             return "No test results found";
         }
 
-        if (summary.getPassCount() == 0) {
```

Comment:
The real issue was that if no test succeeds it is treated as "not tests run". I think we could directly switch to implementation used in JUnit (this implementation is inspired by it) that now also shows the number of failed Tests... https://github.com/jenkinsci/junit-plugin/blob/cf492e95aab0e2e411f5eb7613f9050daf567e42/src/main/java/io/jenkins/plugins/junit/checks/JUnitChecksPublisher.java#L118

Reply:
I agree, thia pr should update extract title method from JUnit plugin

Reply:
@JoC0de, the problem is that other SCMs shows the check in different format (see [PR 97](https://github.com/jenkinsci/xunit-plugin/pull/97)).
It is expected that the "title" shows a brief description, the "summary" shows a resume of the action, and more information, when needed, is shown in the extra info section.
Gitea, for example, doesn't have the summary and the extra info section.
I only use this plugin with GitHub and Gitea, so I can say about other SCMs that integrates with check-api-plugin.
And probably JUnit suffers the same problem (that this one had) that shows the link to send feedback to that plugin in other projects. I really don't want a link to send feedback to other project in a Check in my project.

Reply:
I only refer to the `extractChecksTitle` method. Ok so some examples:
 | xunit title | with JUnit's title | xunit summary  | 
| ------------- | ------------- | ------------- |
 | "There were test failures" | "failed: 2, skipped: 2, passed: 100" | "total:  104, failed: 2, skipped: 2, passed: 100" | 
 | "There were no test executions" | "skipped: 2" | "total:  2, skipped: 2" | 
 | "All tests passed" | "skipped: 2, passed: 100" | "total:  102, skipped: 2, passed: 100" | 
For me both are valid formats I only thought that the extra information is useful, but it is already in the summary (witch is relatively small in GitHub).

Reply:
Isn't the real problem in: https://github.com/jenkinsci/gitea-checks-plugin/blob/0672c320ee47ee3b55ba22f9b80b1d1aa433fd61/src/main/java/io/jenkins/plugins/checks/gitea/GiteaChecksDetails.java#L116
The GitHub and the Gitea "checks view" nearly looks the same (only gitea has no "details" view).
![image](https://user-images.githubusercontent.com/53140583/197579691-56e890dc-ae5a-4426-91f5-25f9ad46b487.png)
![image](https://user-images.githubusercontent.com/53140583/197593851-f7565b14-ac76-46d9-9674-b5cb63c49ab4.png)

Reply:
@JoC0de I will clone `gitea-checks-plugin` and try some changes there to see if it will works with other plugins that I have that are compatible with `checks-api-plugin`. I think all plugins that I have works the way I said early (using summary instead of the title).
If all works good, I'll PR there and the owner of this plugin can revert this PR along PR-97 too.